tests/MongoDB.Driver.Tests/UpdateDefinitionBuilderTests.cs

```diff
@@ -253,16 +265,18 @@ public void Indexed_Typed()
             Assert(subject.Set(x => x.Pets.ElementAt(index).Name, "Fluffencutters"), "{$set: {'pets.2.name': 'Fluffencutters'}}");
         }
 
+
         [Fact]
-        public void Indexed_Positional_Typed()
+        public void Indexed_Typed_Incorrectly()
```

Comment:
It's not the type that's incorrect, it's the value right?

Reply:
Done